### ISSUE TITLE
Video not  covering whole screen

### ISSUE THREAD

**Issue by srinath116** [AgoraIO-Community/ReactNative-UIKit#67]
**Target device:** (Note: simulators are not supported for running the UIKit)
 - Device: Redmi note 9 pro max
 - OS:
   - [] IOS
   - [X] Android
 - OS Version: [e.g. 29]

**App Info** 
 - Type:
   - [X] React Native CLI
   - [ ] Expo CLI (using custom development clients)
 - CLI Version: [e.g. .63.3]

**Describe the issue**
A clear and concise description of what the issue is.
 i have used agora kit in absolute positioned view and the  local video component is not filling the screen for pinned view 

**To Reproduce**
create a new project add a animated view with absolute positioning  , import agora ui kit with default props

**Screenshots**
attahced screenshots of minimized mode and maximised mode for reference
![WhatsApp Image 2022-05-25 at 7 10 42 PM (1)](https://user-images.githubusercontent.com/26498975/170277130-2e2f8249-4228-4c8d-bcfb-86361b6de662.jpeg)
![WhatsApp Image 2022-05-25 at 7 10 42 PM](https://user-images.githubusercontent.com/26498975/170277139-af261927-1387-4001-bee6-157a0f5bdcc8.jpeg)

**Comment by EkaanshArora** [comment]
Can you add some sample code? With absolute positioning you either have to declare a parent view with `position: relative` to set as the container size, or explicitly define the width and height of the absolute view.  
This isn't an issue with the UIKit, you'll need to accommodate the view in your layout. If you want to change the container style of the UIKit to add height/width, you can pass the style to `UIKitContainer` in the `styleProps` prop.

**Comment by srinath116** [comment]
I have applied width and height explicitly for the absolute view ,already  but it is still not occupying the entire screen , even i tried the same with a modal but itz still d same

**Comment by srinath116** [comment]
@EkaanshArora  i have tried both the suggested solutions but it is not working as expected , i even tried it with seperate screen but still its same

**Comment by EkaanshArora** [comment]
Are you able to render a simple absolute <View /> as you want?

**Comment by srinath116** [comment]
@EkaanshArora  yes i was able to  do if u can see the reference images the parent view occupied entire screen , but video container is not filling entire view
